### DISCUSSION TITLE
Http get response doesn't update

### DISCUSSION THREAD

**Discussion by Elphyn** [cc-tweaked/CC-Tweaked#1815] [Q&A] [answered]
I'm not sure, I created a program that uses gist to download and update code, and my update works kinda strange, I haven't found root of the problem yet, is it possible that in computer craft when you do a http get, if you did it sometime ago it would return already saved value and not do an actual request?

**Comment by SquidDev** [comment, marked as answer]
GitHub tends to cache content on their end, so changes to files sometimes take a few minutes to show up (especially on `githubusercontent.com`). The easiest workaround here is to add a cachebuster to the end of the URL, so instead of fetching `"https://gist.githubusercontent.com/x/y/raw/file.lua"`, you download `"https://gist.githubusercontent.com/x/y/raw/file.lua?cb=" .. ("%x"):format(math.random(2 ^ 30))`

**Comment by Elphyn** [comment]
Haha, just figured it out myself, came here to close it, thanks tho!

**Reply by Elphyn** [reply]
Thanks for a workaround!
